src/components/tiles/AddressTile2.tsx

```diff
@@ -232,6 +263,12 @@ export class AddressTileComponent extends React.PureComponent<Props, State> {
                 <FontAwesome name="edit" size={theme.rem(2)} color={theme.iconTappable} />
                 <EdgeText style={styles.buttonText}>{s.strings.enter_as_in_enter_address_with_keyboard}</EdgeText>
               </TouchableOpacity>
+              {canSelfTransfer && (
```

Comment:
We are trying to avoid the `&&` syntax for this case, since we want to enable the strict-boolean lint rule eventually. Use the `?:` syntax, like `canSelfTransfer ? < ... /> : null `